src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+    background-position: center;
+    background-image: url(/src/images/star.svg);
+  }
+
+  &--4 &__star:nth-child(-n + 4) {
+    background-image: url(/src/images/star-active.svg);
+  }
```

Comment:
You have hard coded the active star class for 4 stars. This is a code quality issue. It would be better to use a loop to generate the active star classes for each number of stars. This way, you can easily add or remove active star classes without having to manually code each one.